### ISSUE TITLE
Handle disabled upload speed check correctly

### ISSUE THREAD

**Issue by jgeudens** [pluyckx/kam#11]
When the upload_speed parameter is commented out, kam won't start with following message
```
  File "/usr/sbin/kamd", line 175, in <module>
    raise ex
  File "/usr/sbin/kamd", line 169, in <module>
    main()
  File "/usr/sbin/kamd", line 161, in main
    raise ex
  File "/usr/sbin/kamd", line 154, in main
    check.check()
  File "/usr/local/lib/python3.4/dist-packages/kam/modules/plugins/checks/basecheck.py", line 38, in check
    self._run()
  File "/usr/local/lib/python3.4/dist-packages/kam/modules/plugins/checks/networkspeed.py", line 78, in _run
    "upload_speed", up, "", up >= self._up)
TypeError: unorderable types: float() >= str()
```

**Comment by pluyckx** [comment]
Workaround: do not commemt out and use a high value. Fix will come in next beta version.

**Comment by pluyckx** [comment]
Bug fixed in commit a3d8ed9585b79fd2b15620ac6ebdba1a54a242d3 with patch of jgeudens and a commit to correct some leading spacing. At the moment only available in the dev branch, when other issues are solved, this is also available in version 1.1.0-beta2.
